pkg/client/client.go

```diff
@@ -95,6 +95,16 @@ func (c *kafkaSourceClient) GetKafkaSource(name string) (*v1alpha1.KafkaSource,
 	return kafkaSource, nil
 }
 
+//GetKafkaSources is used to get all available instance of KafkaSource
+func (c *kafkaSourceClient) GetKafkaSources() (*v1alpha1.KafkaSourceList, error) {
```

Comment:
Can you take a look at those `Codecov` annotations and add a few tests to make a bit happier? The client should be pretty straightforward. :)